src/state/user.js

```diff
@@ -24,6 +26,21 @@ const loadUser = async () => {
   return user;
 };
 
+const loadGuidedTours = async () => {
+  const { accessToken } = useAuth();
+  const guidedTour = await PlatformService.loadGuidedTours(accessToken.value);
+  return guidedTour.json();
+};
+
+const completedTour = async tour => {
+  const { accessToken } = useAuth();
+  const guidedTour = await PlatformService.completedTour(
+    accessToken.value,
+    tour
+  );
+  return guidedTour.json();
```

Comment:
I think the `.json()` deserialization should be done in the service rather than the state.